### PR TITLE
1373 - add tests for SlideUp component

### LINKED ISSUE #1373: UNIT TESTS - components/SlideUp
## What

Create unit tests using [Jest](https://jestjs.io/en/) and [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/).

Here are a couple examples of how unit tests should be created:

1. [Unit tests for our A component](https://github.com/thenewboston-developers/Website/blob/development/src/components/A/A.test.tsx)
2. [Unit tests for Icon component](https://github.com/thenewboston-developers/ui-js/blob/master/src/components/Icon/Icon.test.tsx)
3. [Unit tests for Button component](https://github.com/thenewboston-developers/ui-js/blob/master/src/components/Inputs/Button/Button.test.tsx)

## General Acceptance Criteria for writing Unit tests

1. Should create a `[nameOfComponent].test.tsx` within the same directory of the component.
2. Should have tests that tests every `prop` of the component.
3. If there are any conditional logic within the component, there should be unit tests that accounts for each conditional branches.
4. `npm run test` should be passing.

## Specific Acceptance Criteria

1. make sure you tests for each prop

### PR DESCRIPTION
**Pull request by PatNeedham**
closes #1373 

image

image

The three lines at the beginning of the test file setting up the div#slide-up-root element are copied from [the docs](https://testing-library.com/docs/example-react-modal/) (changing the div's ID)

### CODE REVIEW

File: src/components/SlideUp/SlideUp.test.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+describe('SlideUp component', () => {
+  test('renders with text passed in', () => {
+    render(<SlideUp close={() => {}}>Hello World</SlideUp>);
+    expect(screen.getByText('Hello World')).toBeTruthy();
+  });
+
+  test('calls close prop when component is clicked', () => {
+    const close = jest.fn();
+    const {getByRole} = render(<SlideUp close={close}>Hello World</SlideUp>);
```
Comment (angle943) [inline src/components/SlideUp/SlideUp.test.tsx:19]: lets not do this destructuring from render, and grabbing by the Role. You can pass an `data-testid` if you want to grab the button.

**Review by angle943 — CHANGES_REQUESTED** [review]
Can you add tests for when passing along className? Make sure the `bemify(...)` part gets tested too

File: src/components/SlideUp/SlideUp.test.tsx (hunk 1 above)
Comment (PatNeedham) [reply src/components/SlideUp/SlideUp.test.tsx:19]: Thank you for the suggestion. I'm changing it to that in the next git push arriving shortly.

**Comment by angle943** [comment]
I've updated the `data-testid`'s to have parity with the classNames

**Pull request merged**